packages/itwin/property-grid/src/PropertyGridUiItemsProvider.tsx

```diff
@@ -23,14 +23,45 @@ import type { UiItemsProvider, Widget } from "@itwin/appui-react";
 import type { PropertyGridComponentProps } from "./PropertyGridComponent";
 
 /**
- * Id of the property grid widget created by `PropertyGridUiItemsProvider`.
+ * Props for `createPropertyGrid`.
+ * @public
+ */
+ interface PropertyGridProps {
+  /** Widget priority in the stage panel. */
+  defaultPanelWidgetPriority?: number;
```

Comment:
I don't think is should take any props related to placement in panels. Everything can be overridden by consumer.

Reply:
I think there is no need for `PropertyGridProps` any more. `createPropertyGrid` can take `PropertyGridWidgetProps`

Reply:
Changed